src/components/Investigaciones/Investigaciones.jsx

```diff
@@ -1,62 +1,103 @@
 import styles from "./investigaciones.module.css";
 import Temas from '../Temas';
+import { Link } from "react-router-dom";
+import Icons from "../iconos/Icons";
 
-
-const tituloNotaPrincipal = "Titulo de la ultima invtegicacion con un largo máximo de esa cantidad de caracteres: esta cantidad.";
+const tituloNotaPrincipal = "La bonaerense en la mira: de una internación psiquiátrica a un doble femicidio";
     const bajadaNotaPrincipal = "Bajada de la ultima nota que se haya hecho donde el texto puede durar solamente esta cantidad máxima de caracteres";
     const fechaUltimoReporte = "xx/xx/xxxx";
-
+    const fotoNotaPrincipal = "../src/components/investigaciones/investigacion/Crisis_salud_Mental_Mapa3.jpg";
+    const fechaNotaPrincipal = "08/05/2024";
+    const autorxsNotaPrincipal = "Sofi y Pampa";
+    const ilusNotaPrincipal = "Panchopepe";
```

Comment:
are our variables names in english or spanish ?
(i think english)
pick one and stick to it.